west.yml

```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nordic
-      revision: 119ff5b5ec7a413ca2f64ae1928c79eee1e7b7b2
+      revision: 37c38a471c042d890180cfb79cf788a32a006b80
```

Comment:
Upstream PR #: 89489 is outdated and will never be merged. Also why do we need two commit to changed the same version? Can you cherry-pick https://github.com/nrfconnect/sdk-zephyr/pull/2817/commits/d463ea1d10fb1d5c6282a59dcc8256a179fdbae5 from the upmerge PR?

Reply:
need 2 commits because last commit will not apply without conflicts. Cherry-picked newer commit.